tests/FakeItEasy.Tests.ruleset

```diff
@@ -21,4 +22,4 @@
     <Rule Id="CA2000" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
Looks like it... I assume that's because of the ruleset designer; editorconfig only works when you edit the source (which is what I did)